deps.sh

```diff
@@ -54,6 +54,7 @@ then
   pushd ${DEPSDIR}/QCBOR
   cmake -S . -B ${BUILDDIR}/QCBOR \
     -DCMAKE_BUILD_TYPE=Debug \
+    -DCMAKE_C_FLAGS="-fPIC " \
```

Comment:
This actually points to maybe a deeper issue in the upstream config. This could be a workaround for us though...

Reply:
I see what you mean. I was looking at upstream's `Makefile`:

```
CFLAGS=$(CMD_LINE) -I inc -I test -Os -fPIC
```

However they also provide a `CMakeLists.txt` which only calls out `-fPIC` for shared libraries.

Reply:
It looks like there is some difference in defaults on RHEL, which I made explicit option on the other branch. I'll approve this now and can work it out in the other branch.